src/mongocrypt-cache.c

```diff
@@ -242,6 +283,8 @@ _mongocrypt_cache_cleanup (_mongocrypt_cache_t *cache)
 {
    _mongocrypt_cache_pair_t *pair, *tmp;
 
+   BSON_ASSERT_PARAM (cache);
```

Comment:
Suggest `if (!cache) { return; }` for consistency with other cleanup functions.

Reply:
:+1: Change coming in next commit.